app/views/layouts/_footer.html.erb

```diff
@@ -1,8 +1,6 @@
-<footer>
-  <div id="info-bar">
-    <div class="container clearfix">
-      <span class="all-tutorials">Copyright © 2018. Designed by <strong>Trần Đức Trung</strong>. All rights reseved</span>
-      <span class="back-to-tutorial">Logo fb, insta...</span>
-    </div>
+<footer class="py-5 bg-dark">
+  <div class="container">
+    <p class="m-0 text-center text-white">Copyright &copy; 2018. Designed by <a href="#">Trần Đức Trung</a>. All rights reseved</p>
```

Comment:
I18n
tham khảo link này:
http://guides.rubyonrails.org/i18n.html#overview-of-the-i18n-api-features